onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -225,13 +225,48 @@ def aten_col2im(
 ) -> TReal:
     """col2im(Tensor self, SymInt[2] output_size, int[2] kernel_size, int[2] dilation, int[2] padding, int[2] stride) -> Tensor"""
 
+    # Torch prefer to use single number x for kerne,stride,pad,dilation on both side implicitly
+    # But ONNX needs pair number [x,y] to specify on each side explicitly
+    expand_size = 2
+
+    # The dilations should be [x, y]
+    if isinstance(dilation, int):  # x -> [x, x]
```

Comment:
Should we put dilation,padding,stride into kwargs? Because op.Col2im() need them as attribute.

Reply:
Change dilation,padding,stride to attribute. Because if the dilation = (1,1), it will be convert to tensor([1,1]), then op.Col2im(... dilations=dilation...) will failed in FullGraph mode.